cmse/README.md

```diff
@@ -0,0 +1,42 @@
+<!-- SPDX-FileCopyrightText: Copyright 2021 Arm Limited and/or its affiliates <open-source-office@arm.com> -->
+<!-- CC-BY-SA-4.0 AND Apache-Patent-License -->
+<!-- See LICENSE.md file for details -->
+<div align="center">
+   <img src="Arm_logo_blue_RGB.svg" />
+</div>
+
+# Arm MVE Intrinsics Reference
```

Comment:
This seems like the wrong header?

Reply:
Thank you for raising this @pthariensflame .

May I add you to the list of contributors at https://github.com/ARM-software/acle#contributors- for your review?